cloudify-manager-worker/values.yaml

```diff
@@ -306,3 +332,161 @@ containerSecurityContext:
       - AUDIT_WRITE
+
+# -- Parameters greoup for bitnami/rabbitmq helm chart.
+# Details: https://github.com/bitnami/charts/blob/main/bitnami/rabbitmq/README.md
+# @default -- object
+rabbitmq:
+  deploy: true
+
+  fullnameOverride: *rabbitmqFullName
+
+  image:
+    tag: 3.8.9-debian-10-r37
+
+  auth:
+    username: *rabbitmqUsername
+    password: *rabbitmqPassword
+
+    tls:
+      enabled: true
+      existingSecret: *tlsSecretName
+      failIfNoPeerCert: false
+      caCertificate: |-
+      serverCertificate: |-
+      serverKey: |-
+
+  configuration: |-
+    ## Username and password
+    default_user = {{ .Values.auth.username }}
+    default_pass = CHANGEME
+    ## Clustering
+    cluster_formation.peer_discovery_backend  = rabbit_peer_discovery_k8s
+    cluster_formation.k8s.host = kubernetes.default.svc.{{ .Values.clusterDomain }}
+    cluster_formation.node_cleanup.interval = 10
+    cluster_formation.node_cleanup.only_log_warning = true
+    cluster_partition_handling = autoheal
+    # queue master locator
+    queue_master_locator = min-masters
+    # enable guest user
+    loopback_users.guest = false
+    {{ tpl .Values.extraConfiguration . }}
+    {{- if .Values.auth.tls.enabled }}
+    ssl_options.verify = {{ .Values.auth.tls.sslOptionsVerify }}
+    listeners.ssl.default = {{ .Values.service.tlsPort }}
+    ssl_options.fail_if_no_peer_cert = {{ .Values.auth.tls.failIfNoPeerCert }}
+    ssl_options.cacertfile = /opt/bitnami/rabbitmq/certs/ca_certificate.pem
+    ssl_options.certfile = /opt/bitnami/rabbitmq/certs/server_certificate.pem
+    ssl_options.keyfile = /opt/bitnami/rabbitmq/certs/server_key.pem
+    {{- end }}
+    {{- if .Values.ldap.enabled }}
+    auth_backends.1 = rabbit_auth_backend_ldap
+    auth_backends.2 = internal
+    {{- range $index, $server := .Values.ldap.servers }}
+    auth_ldap.servers.{{ add $index 1 }} = {{ $server }}
+    {{- end }}
+    auth_ldap.port = {{ .Values.ldap.port }}
+    auth_ldap.user_dn_pattern = {{ .Values.ldap.user_dn_pattern  }}
+    {{- if .Values.ldap.tls.enabled }}
+    auth_ldap.use_ssl = true
+    {{- end }}
+    {{- end }}
+    {{- if .Values.metrics.enabled }}
+    ## Prometheus metrics
+    prometheus.tcp.port = 9419
+    {{- end }}
+    {{- if .Values.memoryHighWatermark.enabled }}
+    ## Memory Threshold
+    total_memory_available_override_value = {{ include "rabbitmq.toBytes" .Values.resources.limits.memory }}
+    vm_memory_high_watermark.{{ .Values.memoryHighWatermark.type }} = {{ .Values.memoryHighWatermark.value }}
+    {{- end }}
+    management.ssl.port       = 15671
+    management.ssl.cacertfile = /opt/bitnami/rabbitmq/certs/ca_certificate.pem
+    management.ssl.certfile   = /opt/bitnami/rabbitmq/certs/server_certificate.pem
+    management.ssl.keyfile    = /opt/bitnami/rabbitmq/certs/server_key.pem
+
+  resources:
+    limits:
+      memory: 512Mi
+      cpu: 500m
+    requests:
+      memory: 256Mi
+      cpu: 250m
+
+  persistence:
+    enabled: false
```

Comment:
@yuvlar In the old examples (external/rabbitmq.yaml) persistence was disabled by default (so rabbit and postgres was deployed without PV if we using examples from out documentation). Should I leave this behaviour by default or enable PV?

Reply:
The rabbitmq simply holds the queue, I don't think its role requires a PV out-of-the-box unless core/Alex say otherwise. As for postgres, yes, it should have persistence by default for now (until we move to shared RDS, or something similar)

Reply:
Rabbit absolutely needs persistence, yes. I think just /var/lib/rabbitmq should be fine, and rabbit will know how to start up with that populated I believe.

Reply:
persistence enabled for both PostgreSQL and rabbitmq